includes/polling/mempools/bdcom.inc.php

```diff
@@ -1,26 +1,20 @@
 <?php
 /*
+ * LibreNMS
  *
- * LibreNMS mempools polling module for BDCom switches
- *
+ * Copyright (c) 2020 hartred <tumanov@asarta.ru>
  * This program is free software: you can redistribute it and/or modify it
  * under the terms of the GNU General Public License as published by the
  * Free Software Foundation, either version 3 of the License, or (at your
  * option) any later version.  Please see LICENSE.txt at the top level of
  * the source code distribution for details.
-
- * @package    LibreNMS
- * @subpackage polling
- * @link       http://librenms.org
- * @copyright  2017 Carlos A. Pedreros Lizama
- * @author     Carlos A. Pedreros Lizama <carlos.pedreros@gmail.com>
  */
 
-echo 'BDCOM Memory Pool';
-
-$memory_pool = snmp_get_multi_oid($device, ['bdcomMemoryPoolUsed.0', 'bdcomMemoryPoolFree.0'], '-OQUs', 'BDCOM-MEMORY-POOL-MIB');
+$perc = snmp_get($device, ['.1.3.6.1.4.1.3320.3.6.10.1.12.0'], '-OQv', 'NMS-CHASSIS');
 
-$mempool['free']  = ['bdcomMemoryPoolFree.0'];
-$mempool['used']  = $memory_pool['bdcomMemoryPoolUsed.0'];
-$mempool['total'] = $mempool['free'] + $mempool['used'];
-$mempool['perc']  = $mempool['used'] / $mempool['total'];
+if (is_numeric($perc)) {
+    $mempool['perc'] = $perc;
+    $mempool['used'] = $perc;
+    $mempool['total'] = 100;
```

Comment:
memory is meant to be in bytes, not percent...  This will show up as 100 bytes of memory.

Reply:
bdcom >2019year
#sh mem-util 
 mem util threshold is 90%
 current mem util is 27%
 Status:Normal

Reply:
You sure there is not a some OID containing the value in bytes?

Reply:
yes. bdcom changed the policy, and now all their systems are brought to the same form. This data is also displayed in the original monitoring software.

Reply:
@hrtrd Not a single place (inventory, entity-mib, etc etc) where an absolute value for the total memory would be provided ?

Reply:
![Screenshot_1356](https://user-images.githubusercontent.com/20502851/93571390-09ee1100-f99d-11ea-8ebe-28bdf528f038.png)
only this (screenshot). Show usage in percent.  the vendor switched to the new mib libraries. We use now in / opt / librenms / mibs / bdcom already deprecated